disasm/disasm.cc

```diff
@@ -2233,38 +2233,43 @@ void disassembler_t::add_instructions(const isa_parser_t* isa)
     DEFINE_R1TYPE(sm3p1);
   }
 
-  if (isa->extension_enabled(EXT_ZVBB)) {
+  if (isa->extension_enabled(EXT_ZVKB) || isa->extension_enabled(EXT_ZVBB)) {
```

Comment:
Removing redundant check for `EXT_ZVBB` for instructions shared with Zvkb
```suggestion
  if (isa->extension_enabled(EXT_ZVKB)) {
```